.github/workflows/ci.yml

```diff
@@ -53,27 +53,36 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ~1.17
+      id: go
+
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
     - name: Lint
       uses: golangci/golangci-lint-action@v2
```

Comment:
If we were to update to `golangci/golangci-lint-action@v3` it would use the existing installed go version that was setup with the `actions/setup-go@v2`

I think it may be worth it to upgrade this so that we wouldn't have to handle updating the `with.version`. What do you think?